tree/dataframe/src/RDFActionHelpers.cxx

```diff
@@ -133,21 +133,33 @@ BufferedFillHelper::Exec(unsigned int, const std::vector<unsigned int> &, const
 // template void MaxHelper::Exec(unsigned int, const std::vector<unsigned int> &);
 
 MeanHelper::MeanHelper(const std::shared_ptr<double> &meanVPtr, const unsigned int nSlots)
-   : fResultMean(meanVPtr), fCounts(nSlots, 0), fSums(nSlots, 0), fPartialMeans(nSlots)
+   : fResultMean(meanVPtr), fCounts(nSlots, 0), fSums(nSlots, 0), fPartialMeans(nSlots), fCompensations(nSlots)
```

Comment:
ooooo problem might be here! `fCompensations` are not initialized!

Reply:
that's not right, `std::vector` default-initializes its contents: https://en.cppreference.com/w/cpp/container/vector/vector (constructor number 4) and https://godbolt.org/z/ePjshzMnh .

there have been no failures in the last few builds, I think things are okay.